step_definitions/createTeam-steps.js

```diff
@@ -35,7 +35,7 @@ Given(/^teacher successfully logged in$/, async function() {
     return createTeam.createTeamAndGoToSettings(teamname);
   });
 When(/^teacher adds two students to this team$/, function() {
-  return createTeam.addTwoTeamMemebers(fullname1, fullname2)
+  return createTeam.addTwoTeamMemebers('Paula Meyer', fullname2)
```

Comment:
Maybe add a comment here, that this can be extended in the future, when we have a second student perminantly from the beginning?

Reply:
- you can use other credentials of teacher "lehrer@schul-cloud.org" and there we have two students 
- avoid using 'Paula Meyer' better put variable as a parameter in function

Reply:
I agree that is a better way to use a parameter.

Reply:
done